.github/workflows/workflow.yml

```diff
@@ -37,7 +41,77 @@ jobs:
           ruby-version: 2.7.2
           bundler-cache: true
 
-      - run: bundle exec fastlane ci
+      - run: |
+          bundle exec fastlane ci
+          cp .lint/cocoapods-size/result.json cocoapods-size-result.json
+
+      - if: github.event_name == 'push'
+        uses: actions/cache@v3
+        with:
+          path: cocoapods-size-result.json
+          key: cocoapods-size-${{ github.sha }}
+
+  size-report:
+    name: Cocoapods size report
+    if: github.event_name == 'pull_request'
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/cache@v3
+        with:
+          path: cocoapods-size-result.json
+          key: cocoapods-size-${{ github.sha }}
+          restore-keys: cocoapods-size-
+
+      - run: brew install jq
```

Comment:
Is this necessary? I thought some action images shipped with jq.

Reply:
That's true but there is no guarantee of this in the future. I suggest to keep this line just in case